### ISSUE TITLE
Add is_mturk column to CSV schema

### ISSUE THREAD

**Issue by willfenton** [mlopatka/CANOSP2020#85]
Similar to `is_expert` and `is_sumo`, we should include an `is_mturk` column to our CSV schema.

**Comment by mlopatka** [comment]
has this been resolved?
IIRC we handle the mturk annotators similarly to our own annotations. Perhaps this distinction is less relevant than originally thought?

**Comment by willfenton** [comment]
Yeah I think if we're treating all annotations equally, there's no real need for this column. If we kept working on this it might be interesting to weight annotations differently based on `is_mturk`, `is_sumo`, etc. but this is on the backburner.
